<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJack</class>
 <widget class="QDialog" name="BlackJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>631</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="AI_Grid"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>631</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="User_Grid"/>
  </widget>
  <widget class="QPushButton" name="Hit_Button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hit For a Win</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Draw_Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue WIth the Main Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_about">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rules</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
